data-service/README.md

```diff
@@ -74,6 +74,18 @@ docker build -t data-service .
 docker run -p 4000:4000 data-service
 ```
 
+or run bash script
+
+```
+./docker-build.sh
+```
+
+and from core-banking/devops/docker/ run
+
+```
+docker-compose up -d
+```
```

Comment:
can you also add this in the main readme?
its missing there.
basically how to start all the services on dev.

Reply:
we need a bit more explanation for that :) since we also have to build all containers.

Reply:
Added instruction in root directory README how to run current services and db locally. Please check.